solidity-v1/dashboard/src/components/modal/threshold/WithdrawGrantedTokens.jsx

```diff
@@ -0,0 +1,205 @@
+      .toString()
+  }, [grants])
+
+  const onWithdrawClick = (awaitingPromise) => {
+    releaseTokens(selectedGrant, awaitingPromise)
+  }
+
+  return (
+    <>
+      <ModalHeader>Withdraw</ModalHeader>
+      <ModalBody>
+        <h3 className={"mb-1"}>Withdraw granted tokens</h3>
+        <p className={"mb-2 text-grey-70"}>
+          The following tokens are available to be released and withdrawn from
+          your token grant.
+        </p>
+        <div>
+          <OnlyIf condition={grants.length > 1}>
+            <div className="flex row center space-between">
+              <h4 className="text-grey-70">Withdraw:</h4>
+              <span className="text-grey-70 small">
+                Total:{" "}
+                <TokenAmount
+                  amount={totalReadyToRelease}
+                  token={KEEP}
+                  wrapperClassName="withdraw-granted-tokens__total-ready-to-release-amount"
+                  amountClassName="text-grey-60"
+                  symbolClassName="text-grey-60"
+                />
+              </span>
+            </div>
+          </OnlyIf>
+          <form>
+            <Accordion
+              allowZeroExpanded={grants.length > 1}
+              className={
+                "withdraw-granted-tokens__grants-accordion withdraw-granted-tokens__grants-accordion-container"
+              }
+              preExpanded={grants.length === 0 ? [`grant-${grants[0].id}`] : []}
+            >
+              {grants
+                .slice(0, numberOfGrantsDisplayed)
+                .map((grant) =>
+                  renderGrant(
+                    grant,
+                    grants.length,
+                    selectedGrant,
+                    setSelectedGrant
+                  )
+                )}
+            </Accordion>
+          </form>
+          <OnlyIf condition={grants.length > numberOfGrantsDisplayed}>
+            <Button
+              className="withdraw-granted-tokens__view-more-button"
+              onClick={() =>
+                setNumberOfGrantsDisplayed(
+                  numberOfGrantsDisplayed + DEFAULT_GRANTS_TO_DISPLAY
+                )
+              }
+            >
+              <Icons.Add />
+              &nbsp;View More
+            </Button>
+          </OnlyIf>
+        </div>
+      </ModalBody>
+      <ModalFooter>
+        <SubmitButton
+          className="btn btn-primary btn-lg mr-2"
+          type="submit"
+          onSubmitAction={onWithdrawClick}
+          disabled={grants.length > 1 && !selectedGrant}
+        >
+          withdraw
+        </SubmitButton>
+        <Button className={`btn btn-unstyled text-link`} onClick={onClose}>
+          Cancel
+        </Button>
+      </ModalFooter>
+    </>
+  )
+})
+
+const renderGrant = (
+  grant,
+  totalNumberOfGrants,
+  selectedGrant,
+  setSelectedGrant
+) => {
+  return (
+    <AccordionItem uuid={`grant-${grant.id}`} key={`grant-${grant.id}`}>
```

Comment:
Looks like the `preExpanded` and `uuid` don't work. If there is only one grant the accordion item should be expanded.

![image](https://user-images.githubusercontent.com/57687279/142648252-ba618f45-d3c3-4242-9056-ef8df50bc23b.png)

Reply:
Fixed in cbe1c2b11d33b348e6c82cbc19e6b7719c0f1276